apps/store/src/services/storyblok/storyblok.ts

```diff
@@ -173,12 +177,12 @@ export const initStoryblok = () => {
 
 type StoryOptions = {
   locale: string
-  preview?: boolean
+  version?: 'draft' | 'published'
```

Comment:
It seems those 2 types share `version` field definition in a sense that if we change it in one we'd probably change in the other. With that I _nitly_ suggest explicitly tight them up.

```suggestion
  type StoryOptions = { locale: string } & Pick<StoryblokPreviewData, 'version'> // or StoryblokPreviewData['version'] I guess
```

Reply:
Good point.  I extracted StoryblokVersion type to make it explicit